DEVELOPING.md

```diff
@@ -1,22 +1,90 @@
-## Generate python types from Cosmos-SDK protobuf schemas
+## Development setup
 
-### Fetch Cosmos-SDK protobuf schema files
->NOTE: When updating to a different version of the Cosmos SDK (see the version currently used under `COSMOS_SDK_VERSION` in [Makefile](#Makefile])), you will need to perform this step once to fetch the relevant protobuf schema files:
+The easiest way to get set up for development is to install Python `>=3.7` and `pipenv`, then run the following:
 
 ```bash
-make fetch_proto_schema_source
+  make new_env_dev
+  pipenv shell
 ```
 
->NOTE: For this library to be functional, only the python types generated from protobuf schemas are required, not the schema files themselves.
+## Development commands
+
+There are various makefile commands which are helpful during development. Some of the more prominent ones are listed below:
+
+- For linting:
+  ```bash
+    make lint
+  ```
+- For static analysis:
+  ```bash
+    make mypy
+    make pylint
+  ```
+- For code security analysis:
+  ```bash
+    make security
+  ```
+
+- To run tests:
+  ```bash
+    make test
+  ```
+
+## Generating python types from Cosmos SDK protobuf schemas
+
+This library uses python types which are generated (using [Google's Protocol Buffers](https://developers.google.com/protocol-buffers/) compiler) from protocol buffer schemas in the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
+
```

Comment:
It also includes CosmWasm protobuf stuff

Reply:
Added